dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -41,6 +42,7 @@ func Single(args []string) error {
 	singleCmd.StringVar(&hadoopDistributionFlag, "hadoop-distribution", defaultHadoopClient, "the hadoop distribution to build this Alluxio distribution tarball")
 	singleCmd.BoolVar(&skipUIFlag, "skip-ui", false, fmt.Sprintf("set this flag to skip building the webui. This will speed up the build times "+
 		"but the generated tarball will have no Alluxio WebUI although REST services will still be available."))
+	singleCmd.BoolVar(&skipHelmFlag, "skip-helm", true, fmt.Sprintf("set this flag to skip using Helm to generate YAML templates for K8s deployment scenarios"))
```

Comment:
It might be controversial to default to true, which skips Helm templating. My argument is by skipping the helm step, we don't introduce a new dependency on helm for those who builds Alluxio tarballs using the `generate-tarball` script. What do you think @madanadit @Xenorith

Reply:
skip=true sgtm

Reply:
ultimately we publish the generated yaml's by including them in the docker image. so as part of our publish pipeline, we can set this to false.